menu/serializers.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework import serializers
+
+from product.models import Product
+
+
+class MenuSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Product
+        fields = ('id', 'name', 'description', 'volume', 'measure', 'cost', 'logo')
+
```

Comment:
LocationMenuSerializer
`
{
"location_name": "",
"location_address": "",
"products": [ # products here],
}
`

Reply:
done in view